GameWindow.java

```diff
@@ -662,160 +637,118 @@ private void render() // everything in the game that renders
+		g.drawImage(zbutton, 327, 345, this);
+		g.drawImage(stick, 244, 364, this);
+		g.drawImage(cstick, 307, 397, this); // UHJK
+		
+		
+		if(pause==1) 
+		{
+			g.drawImage(pauseIcon, 290, 200, this);
 		}
+		
+		//Scoring letters
+		
+		g.drawString("SCORE:" + score, 20, 25);
+		
+		// Buttons on the controller
+		abutton2.render(g);
+		bbutton2.render(g);
+		xbutton2.render(g);
+		ybutton2.render(g);
+		lbutton2.render(g);
+		rbutton2.render(g);
+		startbutton2.render(g);
+		zbutton2.render(g);
+		stickDLeft2.render(g);
+		
+		
+		stickDown3.render(g);
+		
+		
+		// Anote.render(g);
 
-		if (state == STATE.PLAY) {
-
-			g.drawImage(currentBackground, 0, 0, getWidth(), getHeight(), this);
-			g.setColor(Color.CYAN);
-			g.drawRect(0, 0, 50, 50);
-			g.setFont(new Font("Arial", Font.BOLD, 20));
-			g.drawString("ESC", 5, 32);
-			g.drawImage(controllerr, 220, 350, this);
-			g.drawImage(abutton, 347, 367, this);
-			g.drawImage(bbutton, 333, 372, this);
-			g.drawImage(xbutton, 359, 356, this);
-			g.drawImage(ybutton, 339, 353, this);
-			g.drawImage(lbutton, 258, 336, this);
-			g.drawImage(rbutton, 348, 336, this);
-			g.drawImage(startbutton, 302, 375, this);
-			g.drawImage(zbutton, 347, 345, this);
-			g.drawImage(stick, 264, 364, this);
-			g.drawImage(cstick, 327, 397, this); // UHJK
-			if (pause == 1) {
-				g.drawImage(pauseIcon, 300, 200, this);
-			}
-			// Buttons on the controller
-			abutton2.render(g);
-			bbutton2.render(g);
-			xbutton2.render(g);
-			ybutton2.render(g);
-			lbutton2.render(g);
-			rbutton2.render(g);
-			startbutton2.render(g);
-			zbutton2.render(g);
-			stickDLeft2.render(g);
-			stickDown3.render(g);
-			// Anote.render(g)'
-
-			// Scoring letters
-			g.setFont(new Font("Arial", Font.BOLD, 30));
-			g.setColor(Color.WHITE);
-			g.drawString("SCORE:" + score, 490, 25);
-
-			if (multi.isSelected()) {
-				BnoteMS.render(g);
-				XnoteMS.render(g);
-				stickDownnoteMS.render(g);
-				stickDownnoteMS2.render(g);
-				stickDownnoteMS3.render(g);
-			}
-
-			// We need to divide the buttons that comprise of each tech into this render, so
-			// that they only are on screen when selected
-			if (FoxWavedash.isSelected()) {
-				YnoteFW.render(g);
-				//System.out.println("Y Note IS RENDERED");
-				RnoteFW.render(g);
-				stickDLeftnoteFW.render(g);
+		//MULTISHINE
+		//here I would do stuff dealing with scoring
 			
-				// keeps track of highscore
-				if (score > highscore) {
-					hs.modifyFile(highscore + " FWD", score + " FWD");
-
-					// recreates the array so the new value can be replaced if it increases by more
-					// than one
-					try {
-						hs = new SaveFile();
-					} catch (FileNotFoundException e) {
-						// TODO Auto-generated catch block
-						e.printStackTrace();
-					}
-
+		
+		// We need to divide the buttons that comprise of each tech into this render, so
+		// that they only are on screen when selected
+		if (FoxWavedash.isSelected()) {

```

Comment:
Once you get the technique array set up properly you should be able to get rid of this section entirely.